tests/lua_script.c

```diff
@@ -6,16 +6,46 @@
 #include <lualib.h>
 
 int
-main(int argc, char** argv)
+simple_val()
+{
+    return 42;
+}
+
+int
+return_val(int val)
+{
+    return val;
+}
+
+int
+do_script_arg(char* script_path)
 {
-    /*assert(argc == 2);*/
     lua_State* L = luaL_newstate();
     luaL_openlibs(L);
 
-    /*luaL_dofile(L, argv[1]);*/
+    luaL_dofile(L, script_path);
+
+    lua_close(L);
+    return 0;
+}
+
+int
+do_script()
+{
+    lua_State* L = luaL_newstate();
+    luaL_openlibs(L);
+
+    /*luaL_dofile(L, script_path);*/
```

Comment:
Commented code.

Reply:
Fixed in d528f99d0231980b76039d7d0f86981dc286e6f2.